bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/CameraConfig.java

```diff
@@ -30,6 +30,7 @@ public class CameraConfig {
     public boolean useToken = true;
     private int onvifMediaProfile;
     private int pollTime;
+    private boolean disableSnapshotAtStartup;
```

Comment:
What issue is this to fix? snapshot cameras do not poll the snapshot unless it is necessary, and cameras which use the ffmpeg methods can turn this on and off using the channel `pollImage`. There is also an open issue at the moment for splitting the channel up into two separate channels as currently the channel does two roles. A channel that can be automated anytime is better than a static thing configuration.

Reply:
My intention was that users like me who don't want the "snapshot" ffmepg background process to be started automatically at system start, can disable it with this configuration item.
Ok, I could do this also with a rule executed at startup with some delay.
And it is actually no bug, but whould be a new feature.
Shall I remove it then again?